frontend/packages/metal3-plugin/src/components/dashboard/index.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { Dashboard, DashboardGrid } from '@console/internal/components/dashboard';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { HealthCard } from './health-card';
+
+export const HostDashboard: React.FC<{ obj: K8sResourceKind}> = ({ obj }) => {
+  const mainCards = [
+    () => <HealthCard obj={obj} />
```

Comment:
The type system seems to hate this line. It seems to think that the `obj` prop isn't allowed. @jtomasek any bright ideas?

Reply:
```Type '{ obj: K8sResourceKind; }' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<Component<Pick<Shared<{ [RESULTS_TYPE.URL]: any; [RESULTS_TYPE.PROMETHEUS]: any; } & { watchURL: (url: any, fetch: any) => WatchURL; stopWatchURL: (url: any) => StopWatchURL; watchPrometheusQuery: (query: any) => WatchPrometheus; stopWatchPrometheusQuery: (query: any) =...'.```

I'm not sure why it's truncated.